org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel.ide/pom.xml

```diff
@@ -55,10 +55,8 @@
 							<overWriteIfNewer>true</overWriteIfNewer>
 							<excludeTransitive>true</excludeTransitive>
 							<excludeArtifactIds>
-								com.ibm.icu,
-								org.apache.ant,
-								org.apache.commons.lang,
-								org.apache.commons.logging,
+								icu4j,
+								commons-logging,
```

Comment:
hmmmm in my tests this was not sufficient.

Reply:
From this comment I assume you only checked the ls jar didn't you?
https://github.com/eclipse/xtext/pull/3137#issuecomment-2282236060

But to me it looks like the maven-dependencies-plugin only copies to the `target/libs` folder whoose content is then installed as dependency and then shaded into the uber jar.
And the shading seems to consider all dependencies (I'm not on expert on this), which again includes e.g. commons-logging.
I have to admit I don't know what the ultimate goal was with the exclusion, but it definitively prevents the copying to the `target/libs` folder. It currently does not prevent the inclusion into the ls jar.

Reply:
yes, but on the base of your xtext/commons-logging branch

Reply:
(start runtime eclipse, create new xtext project)